### PR TITLE
[9.0][ADD] provelo_customization: reports leaves view accessible

### PR DESCRIPTION
**Pull request by vvrossem**
[Task](https://gestion.coopiteasy.be/web#id=3367&view_type=form&model=project.task&menu_id=338&action=479)

**Before:**
Report Menu is accessible only to _HR Manager_

**After:**
Report Menu with only Leaves submenu is also accessible to Employee and Department Manager

### CODE REVIEW

File: provelo_customization/wizard/hr_holidays_summary_department_view.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <menuitem
+            name="Leaves by Department"
+            parent="hr_holidays.menu_hr_holidays_report"
+            action="hr_holidays.action_hr_holidays_summary_dept"
+            id="hr_holidays.menu_account_central_journal"
+            icon="STOCK_PRINT"
+            groups="base.group_hr_manager,base.group_hr_user"/>
```
Comment (robinkeunen) [inline provelo_customization/wizard/hr_holidays_summary_department_view.xml]: All `base.group_hr_manager` users are also `base.group_hr_user`:

``` 
    <record id="group_hr_manager" model="res.groups"> ...
        <field name="implied_ids" eval="[(4, ref('group_hr_user'))]"/> ...
    </record>
```

File: provelo_customization/report/available_holidays_view.xml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record model="ir.actions.act_window" id="hr_holidays.action_hr_available_holidays_report">
+        <field name="name">Leaves Analysis</field>
+        <field name="res_model">hr.holidays</field>
+        <field name="view_type">form</field>
+        <field name="view_mode">graph,pivot,calendar,form</field>
+        <field name="view_id"></field>  <!-- force empty -->
```
Comment (robinkeunen) [inline provelo_customization/report/available_holidays_view.xml]: Why do you need to do this?

**Review by robinkeunen — CHANGES_REQUESTED** [review]

File: provelo_customization/report/available_holidays_view.xml (hunk 2 above)
Comment (vvrossem) [reply provelo_customization/report/available_holidays_view.xml]: It's a copy/paste from the source code where I added an additional `context` key-value.
Source code : 
```xml
        <record model="ir.actions.act_window" id="action_hr_available_holidays_report">
            <field name="name">Leaves Analysis</field>
            <field name="res_model">hr.holidays</field>
            <field name="view_type">form</field>
            <field name="view_mode">graph,pivot,calendar,form</field>
            <field name="view_id"></field>  <!-- force empty -->
            <field name="context">{'search_default_leaves':1}</field>
            <field name="search_view_id" ref="view_hr_holidays_filter"/>
        </record>
```

File: provelo_customization/wizard/hr_holidays_summary_department_view.xml (hunk 1 above)
Comment (vvrossem) [reply provelo_customization/wizard/hr_holidays_summary_department_view.xml]: But `base.group_hr_user` are not also `base.group_hr_manager`... Not sure to understand :p

Comment (robinkeunen) [reply provelo_customization/wizard/hr_holidays_summary_department_view.xml]: That means, if you want to give it to `group_hr_user` and `group_hr_manager`, giving the rights to `group_hr_user` is enough :-)

Comment (vvrossem) [reply provelo_customization/wizard/hr_holidays_summary_department_view.xml]: changed in last commit

File: provelo_customization/report/available_holidays_view.xml (hunk 2 above)
Comment (vvrossem) [reply provelo_customization/report/available_holidays_view.xml]: changed in last commit

**Review by robinkeunen — APPROVED** [review]
LGTM with the last changes. The ideal git way to fix for review comments is to use fixups so that

- it's clear that you're amending your own code
- it's easy to `git rebase -i --autosquash` to only keep correct diff

Next time, ok to leave it like that for this PR :-)

**Pull request merged**